azure-communication-ui/chat/src/main/res/values/azure_communication_ui_chat_strings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+   Copyright (c) Microsoft Corporation. All rights reserved.
+   Licensed under the MIT License.
+  -->
+
+<resources>
+    <string name="azure_communication_ui_chat_title_activity_compose_chat">ComposeChatActivity</string>
+    <string name="azure_communication_ui_chat_action_bar_title">Chat</string>
```

Comment:
are we going to use these strings in Chat UI? we need only for demo app no Ui library

Reply:
For ChatView's topbar, we will need this string according to figma as well.

Reply:
Removed the strings.